api/vector_tiles.py

```diff
@@ -58,7 +58,7 @@ def index(self, zoom, x, y):
         mapdb = cherrypy.request.app.config['DB']['map']
         d = mapdb.tables.style.data
 
-        q = sa.select([d.c.rels, d.c.allshields.label('shields'),
+        q = sa.select([d.c.rels, d.c.allrels, d.c.allshields.label('shields'),
```

Comment:
I wonder if it would be better to compute `allrels` on the fly using the hierarchy table. At least the column shouldn't contain really all relations but non-top ones only. You can still return the list of all relations below by concatenating the two lists.